src/components/Type2020/Blueprint.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Blueprint for Q2 2020 Type transition to new font sizes and styles.
+ *
+ *
+ * Name proposals
+ * --------------------------
+ * Type refers to the current soon to be "legacy" Type components.
+ * We need something to differentiate from our current Type component while transitioning.
+ *   - Type2020 (working prototype in this doc & files)
```

Comment:
`Type2` could work and less characters and not tied to a date which doesn't seem ideal to me

Reply:
I'm secretly hoping we can get it done by the end of 2020 ;)

Type2 is less characters, which is a win for sure. I'm open to changing still.

Reply:
I would hope when we're done we delete `Type2` and things return to just `Type`

Reply:
Yes, I've been in this situation, and it turns out to be hard to get to that place. But, in theory, we'd eventually backfill everywhere and then be able to rename